Bottom: cee13bb3bd36ed9a5c371bfabdec39c9b8793988
Top:    2090d8eae642197aefe20fccbf3466f18db0eabf
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-23 12:01:53 -0400

Refresh of mpc5200-ac97

---

diff --git a/sound/soc/fsl/mpc5200_psc_ac97.c b/sound/soc/fsl/mpc5200_psc_ac97.c
index ba8e114..94570ca 100644
--- a/sound/soc/fsl/mpc5200_psc_ac97.c
+++ b/sound/soc/fsl/mpc5200_psc_ac97.c
@@ -1,7 +1,7 @@
 /*
  * linux/sound/mpc5200-ac97.c -- AC97 support for the Freescale MPC52xx chip.
  *
- * Copyright 2008 Jon Smirl, Digispeaker
+ * Copyright (C) 2009 Jon Smirl, Digispeaker
  * Author: Jon Smirl <jonsmirl@gmail.com>
  *
  * This program is free software; you can redistribute it and/or modify
@@ -9,26 +9,14 @@
  * published by the Free Software Foundation.
  */
 
-#include <linux/init.h>
 #include <linux/module.h>
-#include <linux/interrupt.h>
-#include <linux/device.h>
-#include <linux/delay.h>
 #include <linux/of_device.h>
 #include <linux/of_platform.h>
-#include <linux/dma-mapping.h>
 
-#include <sound/core.h>
 #include <sound/pcm.h>
 #include <sound/pcm_params.h>
-#include <sound/initval.h>
 #include <sound/soc.h>
-#include <sound/soc-of-simple.h>
 
-#include <sysdev/bestcomm/bestcomm.h>
-#include <sysdev/bestcomm/gen_bd.h>
-#include <asm/time.h>
-#include <asm/mpc52xx.h>
 #include <asm/mpc52xx_psc.h>
 
 #include "mpc5200_dma.h"
@@ -377,13 +365,13 @@ reset:
 			psc_ac97_warm_reset(&ac97);
 	}
 
+	/* Go */
 	out_8(&regs->command, MPC52xx_PSC_TX_ENABLE | MPC52xx_PSC_RX_ENABLE);
 
-	/* Go */
 	id1 = psc_ac97_read(&ac97, AC97_VENDOR_ID1);
 	id2 = psc_ac97_read(&ac97, AC97_VENDOR_ID2);
 
-	printk("Codec ID is %04x %04x\n", id1, id2);
+	dev_info(&op->dev, "Codec ID is %04x %04x\n", id1, id2);
 
 	return 0;
 }
